no function or associated item named `update_definitions` found for struct `actix_session::Session` in the current scope

Is there any way of fixing this error?

```
error[E0599]: no function or associated item named `update_definitions` found for struct `actix_session::Session` in the current scope
  --> src/main.rs:57:1
   |
57 | #[api_v2_operation]
   | ^^^^^^^^^^^^^^^^^^^ function or associated item not found in `actix_session::Session`
   |
   = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Actix plugin: `#[api_v2_operation]` macro now supports specifying `consumes`, `produces`, `summary`, `description`
 and `operation_id` in macro.
 - Actix plugin: Support for actix-web 3.0.
+- Actix plugin: Support for actix-session.
 
 ### Changed
 - Actix plugin: Internals of `#[api_v2_operation]` proc macro (long-outstanding technical debt). This now generates operation metadata (on the fly) for each handler, which enables us to tie custom changes to operations easily.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -63,6 +63,7 @@ v2 = ["paperclip-macros/v2", "paperclip-core/v2"]
 
 # Features for implementing traits for dependencies.
 actix-multipart = ["paperclip-core/actix-multipart"]
+actix-session = ["paperclip-core/actix-session"]
 chrono = ["paperclip-core/chrono"]
 rust_decimal = ["paperclip-core/rust_decimal"]
 uuid = ["paperclip-core/uuid"]

--- a/core/Cargo.toml
+++ b/core/Cargo.toml
@@ -13,6 +13,7 @@ repository = "https://github.com/wafflespeanut/paperclip"
 actix-web = { version = ">=2.0,<4", optional = true, default-features = false }
 actix-http = { version = ">=1.0,<3", optional = true }
 actix-multipart = { version = "0", optional = true }
+actix-session = { version = "0", optional = true }
 chrono = { version = "0", optional = true }
 heck = { version = "0.3", optional = true }
 once_cell = "1.4"

--- a/core/src/v2/actix.rs
+++ b/core/src/v2/actix.rs
@@ -277,6 +277,12 @@ impl OperationModifier for actix_multipart::Multipart {
     }
 }
 
+#[cfg(feature = "actix-session")]
+impl OperationModifier for actix_session::Session {
+    fn update_parameter(_op: &mut DefaultOperationRaw) {
+    }
+}
+
 macro_rules! impl_param_extractor ({ $ty:ty => $container:ident } => {
     #[cfg(feature = "nightly")]
     impl<T> Apiv2Schema for $ty {

--- a/core/src/v2/actix.rs
+++ b/core/src/v2/actix.rs
@@ -278,9 +278,7 @@ impl OperationModifier for actix_multipart::Multipart {
 }
 
 #[cfg(feature = "actix-session")]
-impl OperationModifier for actix_session::Session {
-    fn update_parameter(_op: &mut DefaultOperationRaw) {}
-}
+impl OperationModifier for actix_session::Session {}
 
 macro_rules! impl_param_extractor ({ $ty:ty => $container:ident } => {
     #[cfg(feature = "nightly")]

--- a/core/src/v2/actix.rs
+++ b/core/src/v2/actix.rs
@@ -38,7 +38,7 @@ pub trait OperationModifier: Apiv2Schema + Sized {
         update_security::<Self>(op);
     }
 
-    /// Update the security defition map (if needed).
+    /// Update the security definition map (if needed).
     fn update_security_definitions(map: &mut BTreeMap<String, SecurityScheme>) {
         update_security_definitions::<Self>(map);
     }
@@ -278,7 +278,9 @@ impl OperationModifier for actix_multipart::Multipart {
 }
 
 #[cfg(feature = "actix-session")]
-impl OperationModifier for actix_session::Session {}
+impl OperationModifier for actix_session::Session {
+    fn update_definitions(_map: &mut BTreeMap<String, DefaultSchemaRaw>) {}
+}
 
 macro_rules! impl_param_extractor ({ $ty:ty => $container:ident } => {
     #[cfg(feature = "nightly")]

--- a/core/src/v2/actix.rs
+++ b/core/src/v2/actix.rs
@@ -279,8 +279,7 @@ impl OperationModifier for actix_multipart::Multipart {
 
 #[cfg(feature = "actix-session")]
 impl OperationModifier for actix_session::Session {
-    fn update_parameter(_op: &mut DefaultOperationRaw) {
-    }
+    fn update_parameter(_op: &mut DefaultOperationRaw) {}
 }
 
 macro_rules! impl_param_extractor ({ $ty:ty => $container:ident } => {

--- a/core/src/v2/schema.rs
+++ b/core/src/v2/schema.rs
@@ -130,6 +130,9 @@ pub trait TypedData {
 }
 
 macro_rules! impl_type_simple {
+    ($ty:ty) => {
+        impl TypedData for $ty {}
+    };
     ($ty:ty, $dt:expr) => {
         impl TypedData for $ty {
             fn data_type() -> DataType {
@@ -191,11 +194,7 @@ impl_type_simple!(
     DataTypeFormat::Binary
 );
 #[cfg(feature = "actix-session")]
-impl_type_simple!(
-    actix_session::Session,
-    DataType::File,
-    DataTypeFormat::Binary
-);
+impl_type_simple!(actix_session::Session);
 #[cfg(feature = "chrono")]
 impl_type_simple!(
     chrono::NaiveDateTime,

--- a/core/src/v2/schema.rs
+++ b/core/src/v2/schema.rs
@@ -190,6 +190,12 @@ impl_type_simple!(
     DataType::File,
     DataTypeFormat::Binary
 );
+#[cfg(feature = "actix-session")]
+impl_type_simple!(
+    actix_session::Session,
+    DataType::File,
+    DataTypeFormat::Binary
+);
 #[cfg(feature = "chrono")]
 impl_type_simple!(
     chrono::NaiveDateTime,

